ginga/rv/plugins/ChangeHistory.py

```diff
@@ -18,26 +18,24 @@
 New history can be added, but old history cannot be deleted,
 unless the image/channel itself is deleted.
 
-The ``redo()`` method picks up a ``'modified'`` event and displays
+The ``redo()`` method picks up an ``'add-image-info'`` event and displays
 related metadata here. The metadata is obtained as follows::
 
         channel = self.fv.get_channel_info(chname)
         iminfo = channel.get_image_info(imname)
         timestamp = iminfo.time_modified
         description = iminfo.reason_modified  # Optional
 
-While ``'time_modified'`` is automatically added by Ginga,
-``'reason_modified'`` is optional and has be to explicitly set
+Both ``'time_modified'`` and ``'reason_modified'`` have to be explicitly set
 by the calling plugin in the same method that issues the
-``'modified'`` callback, like this::
+``'add-image-info'`` callback, like this::
 
-        # This issues the 'modified' callback and sets the timestamp
+        # This changes the data buffer
         image.set_data(new_data, ...)
-        # Manually add the description
-        chname = self.fv.get_channel_name(self.fitsimage)
-        channel = self.fv.get_channel_info(chname)
-        iminfo = channel.get_image_info(image.get('name'))
-        iminfo.reason_modified = 'Something was done to this image buffer'
+        # Add description for ChangeHistory
+        info = dict(time_modified = datetime.utcnow(),
```

Comment:
I think the spaces around the "=" will trigger the flake8 failure.

Reply:
That is just the docstring, but yes, we should give people a good example. I'll fix it when I get the chance. Thanks!

Reply:
No, in the dict(), right?

Reply:
This is a code snippet *inside* a docstring, that is why `flake8` check is not catching it, but I agree it should be fixed.

Reply:
Sorry, I guess I'm not looking at the full code, just the patch...